tests/conftest.py

```diff
@@ -25,6 +25,9 @@
     SERVER_HTTPS_PORT = 8003
     HTTPS_SERVER_URL = f"https://localhost:{SERVER_HTTPS_PORT}"
 
+SLOW_SERVER_HOST = "localhost"
+SLOW_SERVER_HTTPS_PORT = 8004
```

Comment:
The `slow_server` fixture isn't actually using HTTPS:

```suggestion
SLOW_SERVER_PORT = 8004
```